Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -94,7 +94,8 @@ public SpaceMode DistanceSpaceMode
         // The maximum distance before the button is reset to its initial position when retracting.
         private const float MaxRetractDistanceBeforeReset = 0.0001f;
 
-        private float currentPushDistance = 0.0f;
+        public float currentPushDistance = 0.0f;
```

Comment:
this can be private

Reply:
Made it public per suggestion - it could be useful for the developers who want to leverage current push distance.